charts/premium/authelia/questions.yaml

```diff
@@ -988,6 +988,48 @@ questions:
                       schema:
                         type: boolean
                         default: false
+                    - variable: enforce_pkce
+                      label: "Enforce PKCE "
+                      description: "Changing this value to never is generally discouraged; always or public_clients_only or never."
+                      schema:
+                        type: string
+                        default: "public_clients_only"
+                        enum:
+                          - value: "always"
+                            description: "always"
+                          - value: "public_clients_only"
+                            description: "public_clients_only"
+                          - value: "never"
+                            description: "never"
+                    - variable: cors
+                      label: "OpenID CORS"
+                      schema:
+                        additional_attrs: true
+                        type: dict
+                        attrs:
+                          - variable: enabled
+                            label: "Enabled"
+                            schema:
+                              type: boolean
+                              default: false
+                              show_subquestions_if: true
+                              subquestions:
+                                - variable: allowed_origins
+                                  label: "Allowed CORS Origins"
+                                  schema:
+                                    type: list
+                                    default: []
+                                    items:
+                                      - variable: allowedOriginEntry
+                                        label: "Origin"
+                                        schema:
+                                          type: string
+                                          default: "https://oidc.example.com"
+                                - variable: allowed_origins_from_client_redirect_uris
+                                  label: "Automatically adds the origin portion of all redirect URI's on all clients to the list of allowed_origins"
+                                  schema:
+                                    type: boolean
+                                    default: true
```

Comment:
cors is handled at the traefik level for us

Reply:
Interesting. Let me test that out when I’m back home. I’m running the modified version at home. If it works well with this handled at the traefik level I’m all good with that.

Reply:
i will do some cors testing once https://github.com/truecharts/charts/pull/20306 is merged and see if this is needed in addition to traefik or if traefik CORS work well enough. removing this change from this PR.